_data/api-swagger.yaml

```diff
@@ -560,6 +585,112 @@ definitions:
         }
       }
       ```
+  IPAddress:
+    allOf:
+    - $ref: '#/definitions/Verification'
+    - type: object
+      properties:
+        IPAddress:
+          type: string
+          readOnly: true
+    example: |
+      An IP Address verification response may have the following content:
+      ```json
+      {
+        "Type": "IPAddress",
+        "IPAddress": "198.51.100.42"
+      }
+      ```
+  Kennisnet:
+    allOf:
+    - $ref: '#/definitions/Verification'
+    - type: object
+
+  PhoneNumber:
+    allOf:
+    - $ref: '#/definitions/Verification'
+    - type: object
+      properties:
+        Number:
+          type: string
+          description: |
+            The mobile phone number of the signer.
```

Comment:
Better to add the notice that a country calling code (e.g. +31 or +41) is mandatory.